src/node.cc

```diff
@@ -68,6 +68,7 @@
 #ifdef NODE_ENABLE_VTUNE_PROFILING
 #include "../deps/v8/src/third_party/vtune/v8-vtune.h"
 #endif
+#include "node_large_page.h"
```

Comment:
This should also be conditional on `NODE_ENABLE_LARGE_CODE_PAGES`

Reply:
@refack Addressed in the latest updated PR. Can you elaborate more on your suggestion about readme and license file?